plugins/auth-backend/src/providers/oauth2/index.ts

```diff
@@ -14,5 +14,12 @@
  * limitations under the License.
  */
 
+/**
+ * @public
```

Comment:
these can be deleted and should go directly on the definitions instead, inside ./provider